openassessment/xblock/static/js/src/lms/oa_staff_area.js

```diff
@@ -171,10 +186,27 @@
                     deferred.reject();
                 });
             }
+
             return deferred.promise();
         },
 
         /**
+         * Update the counts of ungraded and checked out assessments.
+         */
+        updateStaffGradeCounts: function() {
+            var view = this;
+            var $staffGradeTab = $('.openassessment__staff-grading', this.element);
+
+            view.server.staffGradeCounts().done(function(html) {
+                $staffGradeTab.find('.staff__grade__status').replaceWith(html);
+            }).fail(function() {
+                $staffGradeTab.find('.staff__grade__status').replaceWith(
+                    '<span class="staff__grade__status">gettext("Error getting number of ungraded assessments")</span>'
```

Comment:
Suggest add "the" for understanding, and change "assessments" to "responses": 
"Error getting the number of ungraded responses."